Tribler/Core/Libtorrent/LibtorrentDownloadImpl.py

```diff
@@ -146,17 +145,26 @@ def __init__(self, session, tdef):
         self.askmoreinfo = False
 
         self.correctedinfoname = u""
+        self.checkpoint_disabled = False
 
     def __str__(self):
-        return "LibtorrentDownloadImpl <name: '%s' hops: %d>" % (self.correctedinfoname, self.get_hops())
+        return "LibtorrentDownloadImpl <name: '%s' hops: %d checkpoint_disabled: %d>" % \
+               (self.correctedinfoname, self.get_hops(), self.checkpoint_disabled)
 
     def __repr__(self):
         return self.__str__()
 
     def get_def(self):
         return self.tdef
 
-    def setup(self, dcfg=None, pstate=None, initialdlstatus=None, wrapperDelay=0):
+    def set_checkpoint_disabled(self, disabled=True):
+        self.checkpoint_disabled = disabled
+
+    def get_checkpoint_disabled(self):
```

Comment:
Do we really want a getter/setter for a public attribute? I suggest we should either remove the getter/setter or make the `checkpoint_disabled` property private (by prefixing it with an underscore).